packages/components/src/test/properties/NumericPropertyInput.test.tsx

```diff
@@ -259,4 +259,35 @@ describe("<NumericInput />", () => {
 
     expect(spy).to.be.calledWith("1e-");
   });
+
+  it("does not fire `onBlur` when value is changed but the input is not blurred", async () => {
+    const user = userEvent.setup();
+    const spy = sinon.spy();
+    const { getByRole } = render(<NumericInput handleBlur={spy} onChange={() => {}} value="2" />);
+    const inputContainer = await waitFor(() => getByRole("textbox"));
+
+    await user.type(inputContainer, "1");
+
+    expect(spy).to.be.not.be.called;
+  });
```

Comment:
I don't think this test is necessary. 

Generally we test something for doing things, unless those things are commonly done and it's unusual to not do them. Calling a blur callback on value change is not a common thing to do, so we don't need to test this.